no/Mortgage/v0/example/MortgageApplicationCreated.json

```diff
@@ -0,0 +1,116 @@
+{
+    "brokerReference": {
+        "issuer": "io.klira",
+        "id": "hc8awe-ja8oJOJ8-ha0Xa2P"
+    },
+    "dataProtectionContext": "FICTIONAL",
+    "application": {
+        "applicant": {
+            "ssn": "12345678901",
+            "phone": "+44123456789",
```

Comment:
```suggestion
            "phone": "+47123456789",
```

Reply:
Okay, I guess Norweigan numbers are more authentic :stuck_out_tongue:

Reply:
(had to go into high level nitpicking to find SOMETHING :laughing:  )